src/pages/MainPage.jsx

```diff
@@ -26,6 +26,11 @@ function MainPage() {
   const [groupTitle, setGroupTitle] = useState('');
   const history = useHistory();
 
+  function logout() {
+    localStorage.clear();
+    history.go('/');
```

Comment:
May want to use `history.push` or `history.replace` here, `history.go` takes a number and goes back in the stack for that many entries